frontends/ast/simplify.cc

```diff
@@ -268,6 +263,106 @@ static AstNode *make_range(int left, int right, bool is_signed = false)
 	return range;
 }
 
+static AstNode *expand_dot(const AstNode* current_struct, const AstNode* search_node) {
+	AstNode *current_struct_elem = nullptr;
+	for (auto c : current_struct->children) {
```

Comment:
Can't this be replaced with a `vector.find`?
`search_str` should be defined before this loop.

Reply:
Done.